projects/Mallard/src/screens/identity-login-screen.tsx

```diff
@@ -103,9 +103,19 @@ const AuthSwitcherScreen = ({
                         }
                     } catch (e) {
                         setIsLoading(false)
-                        setError(
-                            typeof e === 'string' ? e : 'Something went wrong',
-                        )
+                        if (
+                            String(e).includes(
+                                'Error: The Apple authentication request has been canceled by the user',
```

Comment:
I think you should be able to detect the error code rather than doing a string match https://docs.expo.io/versions/latest/sdk/apple-authentication/#error-codes 

and maybe just 'Sign in cancelled' as the user ought to know that they were the person that cancelled it